pkg/intermediate/aggregate.go

```diff
@@ -168,9 +169,23 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 	a.mutex.Lock()
 	defer a.mutex.Unlock()
 
+	correlationRequired := false
+	// Get Antrea flowType from the record that will help to determine if correlation
+	// is required or not.
+	if ieWithValue, exist := record.GetInfoElementWithValue("flowType"); exist {
+		if ieWithValue.Element.DataType == entities.Unsigned8 {
+			recordFlowType := ieWithValue.Value.(uint8)
```

Comment:
These two lines seem redundant. Why not use `if recordFlowType, ok := ieWithValue.Value.(uint8); ok {}`

Reply:
Yes, that should be simpler.